api/src/AppBundle/Entity/Repository/DocumentRepository.php

```diff
@@ -22,4 +25,77 @@ public function retrieveSoftDeleted()
 
         return $records;
     }
+
+    public function getQueuedDocumentsAndSetToInProgress()
+    {
+        $limit = self::ROWS_LIMIT;
+        // Using DENSE_RANK here as we get multiple rows for the same document due to multiple report submissions. This
+        // ensures any limit applied will not miss out submissions by chance
+        $queuedDocumentsQuery = "
+SELECT case_number, document_id, document_report_submission_id, is_report_pdf, filename, storage_reference, report_start_date, report_end_date, report_submit_date, report_type, ndr_id, ndr_start_date, ndr_submit_date, report_submission_id, report_submission_uuid
```

Comment:
SQL looks fine but for readability I dunno if we wouldn't want to put each of the fields on a new line? It might make it quite long but maybe easier for people going forward.

Reply:
Good point - I'll drop them down on to individual lines 👍